bundle/Templating/Twig/IbexaAdminUIExtension.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\BetterIbexaAdminUIBundle\Templating\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class IbexaAdminUIExtension extends AbstractExtension
+{
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('count_by_content_type', [IbexaAdminUIRuntime::class, 'countByContentType']),
```

Comment:
This should be namespaced with `ng_` prefix, also make it more descriptive, for example: `ng_count_content_by_content_type`.

Reply:
fixed in f7d600a28996073357c663c105cdea38049022f3